samples/bluetooth/hci_rpmsg/boards/nrf5340dk_nrf5340_cpunet.conf

```diff
@@ -0,0 +1,32 @@
+CONFIG_BT_BUF_EVT_RX_COUNT=30
+
+CONFIG_BT_BUF_EVT_RX_SIZE=255
+CONFIG_BT_BUF_ACL_RX_SIZE=255
+CONFIG_BT_BUF_ACL_TX_SIZE=251
+CONFIG_BT_BUF_CMD_TX_SIZE=255
+
+CONFIG_BT_EXT_ADV=y
+CONFIG_BT_PER_ADV=y
+CONFIG_BT_PER_ADV_SYNC=y
+
+CONFIG_BT_CTLR_ASSERT_HANDLER=y
+CONFIG_BT_CTLR_DTM_HCI=y
+
+CONFIG_BT_CTLR_RX_BUFFERS=18
+CONFIG_BT_CTLR_DATA_LENGTH_MAX=251
+
+CONFIG_BT_CTLR_PHY_CODED=y
+
+CONFIG_BT_CTLR_ADV_SET=2
+CONFIG_BT_CTLR_ADV_DATA_LEN_MAX=191
+CONFIG_BT_CTLR_SCAN_DATA_LEN_MAX=1650
+
+CONFIG_BT_CTLR_ADVANCED_FEATURES=y
+CONFIG_BT_CTLR_ADV_AUX_SET=2
+CONFIG_BT_CTLR_ADV_SYNC_SET=2
+CONFIG_BT_CTLR_ADV_DATA_BUF_MAX=6
+
+# Increase the below to receive interleaved advertising chains
+# CONFIG_BT_CTLR_SCAN_AUX_SET=2
```

Comment:
Is it possible to avoid using BT_LL_SW_SPLIT specific configurations in these samples? These samples are typically also built for other controllers. It is a bit annoying to get warnings about using undefined kconfigs. 

I would believe it is possible to set the default values to something that makes the controller qualifiable

Reply:
I agree, will see what I can do.